.github/workflows/deploy.yml

```
@@ -29,6 +29,10 @@ jobs:
           git clone https://github.com/open-contracting/ocds-extensions-translations.git
           ocdsextensionregistry generate-data-file --locale-dir ocds-extensions-translations/locale > extension_explorer/data/extensions.json
           python freeze.py
+      - name: Compile translations
+        run: |
+          cd extension_explorer
+          pybabel compile -d translations -f
```

I'm not sure if the `cd` affects later steps. This is safer.

```suggestion
        working-directory: extension-explorer
        run: pybabel compile -d translations -f
```